tools/CreditsTool/remappings.txt

```diff
@@ -18,3 +18,13 @@ PraiseRatvar Frozenguy5
 FuryMcFlurry Fury McFlurry
 vuonojenmustaturska Naksu
 praisenarsie Frozenguy5
+MrDoomBringer Mr. DoomBringer
+Fikou Dr. Fikou
+TiviPlus Tivi Plus
+tralezab Trale Zab
+Iamgoofball goofball
+Tharcoonvagh Tharcoon
+Rectification itseasytosee
+ATHATH ATH1909
+trollbreeder troll breeder
+BuffEngineering Buff Engineering
```

Comment:
Do all of these agree to this change?

Reply:
Yup, I've followed up with them individually on discord about it, everyone's fine with it..